lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -70,11 +82,11 @@ foreach ($member in $members) {
     }
 
     if ($state -in @("present", "pure") -and !$user_in_group) {
-        Add-ADGroupMember -Identity $name -Members $group_member -WhatIf:$check_mode @extra_args
+        Add-ADPrincipalGroupMembership -Identity $group_member -MemberOf $name -WhatIf:$check_mode
```

Comment:
Why the change in cmdlets, also it's now missing`@extra_args` so things like the Server and Credentials no longer apply.

Reply:
i had massive issues with `Add-ADGroupMember`  and all helps and workarounds i found online regarding multi domain support used Add-ADPrincipalGroupMembership.